matching-notification-api/src/main/java/org/phenotips/matchingnotification/storage/internal/DefaultMatchStorageManager.java

```diff
@@ -110,10 +110,14 @@
      * of the match, so need to show only one of them (the notified one).
      * */
     private static final String HQL_QUERY_FIND_ALL_MATCHES_BY_SCORE =
-            "from DefaultPatientMatch as m where"
+        "from DefaultPatientMatch as m where"
             + " score >= :minScore and phenotypeScore >= :phenScore and genotypeScore >= :genScore and"
             + " (m.notified = true or (not exists (" + HQL_QUERY_SAME_PATIENT_BUT_NOTIFIED + ")))";
 
+    /** A query used to get the number of all remote matches. */
+    private static final String HQL_QUERY_REMOTE_MATCHES =
+        "select count(*) from DefaultPatientMatch as m where m.referenceServerId = '' or m.matchedServerId =''";
```

Comment:
this will return a local match which has both of those servers as blank. I think what you mean is `!= ''`